externs/shaka/player.js

```diff
@@ -1829,7 +1829,8 @@ shaka.extern.LiveSyncConfiguration;
  *   If <code>true</code>, the media source live duration
```

Comment:
It's not related to this PR, but I really don't like the description of this config value. It is worded in an awkward way that isn't quite grammatical, and it doesn't say what the behavior is if the value is false. I'd say...

> If <code>true</code>, the media source duration of livestreams will be set to <code>Infinity</code>. Otherwise, it will be set to a high but non-infinite number (2^32).

Reply:
Done!